contracts/token/FundRequestToken.sol

```diff
@@ -1,24 +1,23 @@
 pragma solidity ^0.4.13;
 
 
-import "./MiniMeToken.sol";
+import "./LimitedTransferMiniMeToken.sol";
 
 
 /**
  * Fundrequest Token as deployed by https://fundrequest.io
  * Davy Van Roy
  * Quinten De Swaef
  */
-contract FundRequestToken is MiniMeToken {
+contract FundRequestToken is LimitedTransferMiniMeToken {
```

Comment:
Can we somehow change this to a Minime Token with a TransferCapability? So we can have composition over inheritance? Not necessary, but could be nice